view/vite.config.js

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+export default defineConfig({
+  plugins: [react()],
+  server: {
+    proxy: {
+      '/api': {
+        target: 'http://localhost:8080/api',
```

Comment:
프록시를 세분화해야 하면 될 것 같네용

```suggestion
      ...
      '/api/v1/restaurants': {
        target: 'http://localhost:7002/api/v1/restaurants',
      ...
```